license/src/main/resources/APACHE-2-SIMPLIFIED-COPYRIGHT.txt

```diff
@@ -0,0 +1,13 @@
+Copyright (C) ${project.inceptionYear} ${owner}
```

Comment:
It might also make sense to make this module a standalone project with its own release cycle. It's not going to change much, so releasing with every redhat-releng-tools release doesn't make a lot of sense strictly speaking. Also, we could really use something like this in the Commonjava projects too. I never dug into the license plugin far enough to know how to do this, but it's something that has always bothered me.